src/rest/server_service/path_predicate.rs

```diff
@@ -0,0 +1,26 @@
+use actix_web::pred::Predicate;
+use actix_web::Request;
+use regex::{self, Regex};
+
+pub struct PathPredicate {
+    regex: Regex,
+}
+
+impl PathPredicate {
+    pub fn new(prefix: &str, pattern: &str) -> Self {
+        let full_pattern = format!("^/{}/{}$", regex::escape(prefix), pattern);
```

Comment:
I would rather demand the leading slash to be part of the prefix pattern, for usability reasons.